docs/university/course-three-workshop.md

```diff
@@ -0,0 +1,107 @@
+# Data and Device Management Workshop
+
+## Introduction
+
+Welcome to your first workshop! Along with the videos, we wanted to give you real, step-by-step instructions on how to accomplish and use most of the tools we covered in the videos. So, from here on out, you can expect every course to have a workshop.
+
+The workshops build upon one another. So, as you create devices, workflows, etc. in the workshops, make sure to keep those around as they may come up again in the future.
+
+This workshop is all about Data and Device Management. We are going to use the platform features talked about in Course 3, such as [creating a device](/devices/overview/), the [Device Simulator](/devices/simulator/), and the [Data Explorer](/applications/data-explorer/).
+
+Let’s begin.
+
+## Story
+
+Before we start building, let’s describe what you are building. Imagine you work for a company that manufactures industrial equipment. This company is named Embree Manufacturing.
+
+<div class="slim-image-container"><img style="padding: 10px;" src="/images/university/course-three-workshop/embree-logo.png" alt="Auto Setup Complete" title="Auto Setup Complete" /></div>
+
+Embree is launching an initiative to adopt IoT and offer new capabilities to their customers. Currently, Embree produces water pumps for industrial use cases like mining, construction, and oil & gas. Their goal is to gain deeper insight into the pumps and provide more value to their customers.
+
+To begin the new initiative, the Embree team decided to start with a proof-of-concept, which will enable them to solve a small, useful problem to get results faster. For this first project, Embree wants to take one of their existing pumps and send the data it’s collecting, like flow rate and temperature, to the cloud.
```

Comment:
Not a show-stopper but it looks like the concensus is to capitalize "the Cloud".